### DISCUSSION TITLE
Joystick controller for TTGO VGA32 and Vic-20 emulation

### DISCUSSION THREAD

**Discussion by GameBoyzGB** [fdivitto/FabGL#323] [Q&A]
I would like to connect an Atari joystick to the TTGO VGA32 v1.4 so that I can use it for the Vic-20 games that support joystick only. Hopefully someone can help or has already done it.

Thanks!

**Comment by OzHawk** [comment]
Quite some time ago, I tried to convince Fabrizio to add the interface and
code for (S)NES controllers.  Many more controls, and may be made to appear
as Atari style joysticks in software.
